rj_gameplay/stp/coordinator.py

```diff
@@ -27,16 +27,14 @@ class Coordinator:
 
```

Comment:
This class is now unused, can delete later

Reply:
Why not delete now?